atd-vze/src/views/Locations/LocationMap.js

```diff
@@ -168,12 +171,7 @@ export default class LocationMap extends Component {
         mapboxApiAccessToken={TOKEN}
       >
         {/* add nearmap raster source and style */}
-        {!isDev && (
-          <>
-            <Source {...LOCATION_MAP_CONFIG.sources.aerials} />
-            <Layer {...LOCATION_MAP_CONFIG.layers.aerials} />
-          </>
-        )}
+        {!isDev && <LabeledAerialSourceAndLayer />}
```

Comment:
We didn't show the street labels on top of the NearMap tiles here previously, but I think that we should. What do y'all think?

Reply:
absolutely! i think the street labels are always super helpful :sparkles:

Reply:
i actually am seeing them in staging though on locations?

Reply:
@roseeichelmann thank you for catching this! 🕵️‍♂️ The street layer was in here a few lines below. The new `LabeledAerialSourceAndLayer` component introduced a duplicate so I removed the old one that was on its own. 🙏